src/app/login-signup/sign-up/sign-up.component.html

```diff
@@ -53,28 +131,45 @@
 					[(ngModel)]="confirmPassword"
 					name="ConfirmInput"
 					spellcheck="false"
+					(focus)="showTooltipCP = true"
+					(blur)="showTooltipCP = false"
 				>
 
+				<app-icon
+					*ngIf="password === confirmPassword && validateValuesService.validatePassword(password).correct"
```

Comment:
We don't need to check this password.
```suggestion
					*ngIf="password === confirmPassword"
```